pyfar/dsp/filter/audiofilter.py

```diff
@@ -721,7 +1059,9 @@ def _shelve_cascade(signal, frequency, frequency_type, gain, slope, bandwidth,
         return filt.process(signal), N, ideal_response
 
 
-def _shelving_cascade_slope_parameters(gain, slope, bandwidth, shelve_type):
+@rename_arg({"shelve_type": "shelf_type"}, "shelve_type will be deprecated in"
+            " version 0.9.0 in favor of shelf_type")
+def _shelving_cascade_slope_parameters(gain, slope, bandwidth, shelf_type):
```

Comment:
its a private function, so we dont need to deprecate it, we can change directly. It should just be used within pyfar and not outside